docs/source/deprecated.rst

```diff
@@ -0,0 +1,32 @@
+==========
+Deprecated
+==========
```

Comment:
```suggestion
============
Deprecations
============
```

\+ rename the file

I'm wondering if the module wouldn't be better named `sopel.deprecation`, actually.

Reply:
> + rename the file

What about this?